server/bastionlab_polars/src/composite_plan.rs

```diff
@@ -150,6 +150,14 @@ impl CompositePlan {
     }
 }
 
+impl Deref for CompositePlan {
```

Comment:
This was so I could get an iterator of the CompositePlan for my function which iterates over the composite plan and gets the entrypoints for the inputs column in log because I couldn't get an iter or access to composite_plan.0 from lib.r. I see that I can put that function into CompositePlan instead and call it from lib.rs (and so delete Deref).